doc/spec/manifest.md

```diff
@@ -0,0 +1,241 @@
+        and should be long enough to encode the full digest produced by the
+        corresponding digest algorithm.
+
+        Optionally, the `digest` value may be an absolute reference to a digest
+        in another repository:
+
+            <repository>@<digest-alg-name>:<hex-encoded-digest>
+
+        A `digest` in this form is considered relative to the repository
+        explicitly specified in the `<repository>` section of the reference.
+        The `<repository>` value must match the format specified by the
+        [Namespace and Discovery Documentation](https://github.com/dmcgowan/distribution/blob/namespace-proposal/doc/spec/namespace.md#name-grammar).
+    
+    Any media type may be in the `mediaType` field as the application may
+    use any type of content. If the `mediaType` is
+    `application/vnd.docker.distribution.content.manifest.v2+json` then the
+    dependency is another content manifest as described by this document. The
+    dependencies from that manifest should also be considered dependencies of
+    this application.
+
+    In general, the ordering of these dependencies is significant as it may be
+    significant to the application. However, it is best for the application
+    data below to also designate what actions to perform on the dependencies.
+
+- **`application`** *object*
+
+    Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the JSON object in the `data` field.
+    
+    - **`data`** *object*
+    
+        Abstract application data in any schema, as described by the
+        `mediaType` field.
+    
+    Application data may be *any* JSON object, allowing this manifest format,
+    along with the generic dependencies list, to support any type of 
+    application that can be represented as a JSON configuration and a
+    collection of content-addressable blobs of data.
+
+- **`meta`** *object*
+
+    Meta may be *any* JSON object, allowing content creators to annotate
+    their application manifest with any additional labels beyond those already
+    defined by this manifest format. 
+
+### Example Manifest
+
+```json
+{
+    "schemaVersion": 2,
+    "createdAt": "2015-03-04T17:52:02.841028307-08:00",
+    "maintainers": [
+        {
+            "name": "Phillip Edward Nuñez",
+            "email": "pnunez@example.com",
+            "url": "https://example.com/pnunez/"
+        }
+    ],
+    "description": "Application to Simulate Turbo Encabulators",
+    "keywords": [
+        "foo",
+        "bar",
+        "baz"
+    ],
+    "website": "https://foo.bar.baz.example.com",
+    "licenses": [
+        "Apache-2.0"
+    ],
+    "dependencies": [
+        {
+            "mediaType": "application/vnd.docker.distribution.content.manifest.v2+json",
+            "length": 1024,
+            "digest": "isv.example.com/base/system@sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7"
```

Comment:
would it be needed to have a bullet in the dependencies section around qualified domain docker name? For the federated layers, it would provide insight for this pattern